### PR TITLE
translations PR 20160929

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the [Mattermost Translation Server](http://translate.mattermost.com) and probably will be submitted for the next PR.

### CODE REVIEW

**Comment by chikei** [comment]
`zh_TW` LGTM

File: i18n/de.json [hunk 1]
```diff
@@ -1154,6 +1150,10 @@
     "translation": "Schlechter Dateiname zurückgewiesen, Dateiname=%v"
   },
   {
+    "id": "api.post.create_post.can_not_post_to_deleted.error",
+    "translation": "Kann in gelöschten Kanal senden."
```
Comment (monsdar) [inline i18n/de.json]: "Kann nicht in gelöschten Kanal senden"

File: i18n/de.json [hunk 2]
```diff
@@ -1400,6 +1400,14 @@
     "translation": "Anfragenbegrenzereinstellung nicht korrekt konfiguriert, verwende VaryByHeader und deaktiviere VaryByRemoteAddr"
   },
   {
+    "id": "api.server.start_server.rate_limiting_memory_store",
+    "translation": "Konnte Anfragenbegrenzer nicht initialisieren."
```
Comment (monsdar) [inline i18n/de.json]: Anfragenbegrenzer sounds very "try hard". In my opinion there are certain things like error messages (or single terms in those messages) that better stay English.

File: i18n/de.json [hunk 3]
```diff
@@ -1508,6 +1528,14 @@
     "translation": "Zip Datei konnte nicht geöffnet werden"
   },
   {
+    "id": "api.slackimport.slack_parse_channels.error",
+    "translation": "Fehler beim verarbeiten der Slack Kanäle. Import könnte trotzdem funktionieren."
+  },
+  {
+    "id": "api.slackimport.slack_parse_posts.error",
+    "translation": "Fehler beim Verarbeiten einiger Slach Nachrichten. Import könnte trotzdem funktionieren."
```
Comment (monsdar) [inline i18n/de.json]: *Slack

File: i18n/de.json [hunk 4]
```diff
@@ -2356,10 +2388,34 @@
...
+    "id": "authentication.permissions.team_use_slash_commands.description",
+    "translation": "Die Möglichkeit Slash Befehle zu verwenden"
+  },
+  {
+    "id": "authentication.permissions.team_use_slash_commands.name",
+    "translation": "Slash Befehle verwenden"
+  },
+  {
     "id": "cli.license.critical",
     "translation": "Feature erfordert Enterprise Lizenz. Bitte kontaktieren Sie Ihren Systemadministrator wegen Upgrade Ihrer Enterprise Lizenz."
```
Comment (monsdar) [inline i18n/de.json]: "...zum Upgrade Ihrer Enterprise Lizenz."

Comment (monsdar) [inline i18n/de.json]: Also possible: "...zwecks Upgrade..."

File: i18n/pt-BR.json [hunk 5]
```diff
@@ -1400,6 +1400,14 @@
     "translation": "RateLimitSettings não foi configurado corretamente usando VaryByHeader e desativando VaryByRemoteAddr"
   },
   {
+    "id": "api.server.start_server.rate_limiting_memory_store",
+    "translation": "Não foi possível inicializar o rate limiting."
+  },
+  {
+    "id": "api.server.start_server.rate_limiting_memory_store",
```
Comment (rodrigocorsi2) [inline i18n/pt-BR.json]: Translation id repeated:
"id": "api.server.start_server.rate_limiting_memory_store"

Comment (enahum) [inline i18n/pt-BR.json]: that shouldn't happen, let me try and figure out why is it.

thanks @rodrigocorsi2

File: i18n/pt-BR.json [hunk 6]
```diff
@@ -3076,6 +3144,10 @@
     "translation": "Inválido máxima tentativas de login em configurações de serviço.  Deve ser um número positivo."
   },
   {
+    "id": "model.config.is_valid.max_burst.app_error",
+    "translation": "Max burst must be greater than zero."
```
Comment (rodrigocorsi2) [inline i18n/pt-BR.json]: "translation": "Burst máximo deve ser diferente de zero."

**Comment by enahum** [comment]
@jasonblais can you please take a look at this two in `/platform/i18n/en.json` so I can fix that for the whole PR.

```json
{
    "id": "api.server.start_server.rate_limiting_memory_store",
    "translation": "Unable to initalize rate limiting memory store. Check MemoryStoreSize config setting."
  },
  {
    "id": "api.server.start_server.rate_limiting_memory_store",
    "translation": "Unable to initalize rate limiting."
  },
```

thanks

File: i18n/pt-BR.json (hunk 6 above)
Comment (enahum) [inline i18n/pt-BR.json]: yeap, probably the translation wasn't in the translations server when I submitted the PR, the same goes for the rest of the languages

File: webapp/i18n/pt-BR.json [hunk 7]
```diff
@@ -531,6 +532,9 @@
   "admin.rate.httpHeaderDescription": "Quando preenchido, variam limitação de taxa pelo compo cabeçalho HTTP especificado (ex. quando configurado NGINX ajustado para \"X-Real-IP\", quando configurado AmazonELB ajustado para \"X-Forwarded-For\").",
   "admin.rate.httpHeaderExample": "Ex \"X-Real-IP\", \"X-Forwarded-For\"",
   "admin.rate.httpHeaderTitle": "Variar o rate limit pelo cabeçalho HTTP",
+  "admin.rate.maxBurst": "Max Burst:",
```
Comment (rodrigocorsi2) [inline webapp/i18n/pt-BR.json]: "admin.rate.maxBurst": "Burst Máximo:"

File: i18n/de.json (hunk 4 above)
Comment (enahum) [inline i18n/de.json]: Thanks @monsdar sadly any changes to the translations have to be done in the Translation server, that way all the changes can be included in a future PR.

File: webapp/i18n/pt-BR.json (hunk 7 above)
Comment (enahum) [inline webapp/i18n/pt-BR.json]: same as my comment above :)

File: webapp/i18n/pt-BR.json [hunk 8]
```diff
@@ -531,6 +532,9 @@
...
+  "admin.rate.maxBurstDescription": "Maximum number of requests allowed beyond the per second query limit.",
```
Comment (rodrigocorsi2) [inline webapp/i18n/pt-BR.json]: "admin.rate.maxBurstDescription": "Máximo número de pedidos permitidos além do limite de consultas por segundo."

File: webapp/i18n/pt-BR.json [hunk 9]
```diff
@@ -823,6 +828,29 @@
   "admin.user_item.switchToEmail": "Trocar para Email/Senha",
   "admin.user_item.sysAdmin": "Admin do Sistema",
   "admin.user_item.teamAdmin": "Admin Equipe",
+  "admin.webrtc.enableDescription": "When true, Mattermost allows making <strong>one-on-one</strong> video calls. WebRTC calls are available on Chrome, Firefox and Mattermost Desktop Apps.",
```
Comment (rodrigocorsi2) [inline webapp/i18n/pt-BR.json]: "admin.webrtc.enableDescription" : "Quando verdadeiro, Mattermost permite fazer vídeo chamadas <strong>ponto-a-ponto</strong>. Chamadas WebRTC estão disponíveis no Chrome, Firefox e Mattermost Desktop Apps."

File: i18n/pt-BR.json (hunk 5 above)
Comment (rodrigocorsi2) [inline i18n/pt-BR.json]: :+1:

**Comment by rodrigocorsi2** [comment]
pt-BR seems right to me :+1:

**Comment by jasonblais** [comment]
@enahum: 

"api.server.start_server.rate_limiting_memory_store": `Unable to initialize rate limiting memory store. Check MemoryStoreSize config setting is correct.` [typo + minor tweak]
"api.server.start_server.rate_limiting_memory_store": `Unable to initialize rate limiting.` [typo]

^ I'm not sure what 'memory store' is, so I assume that's correct and not referring to 'memory storage'

**Comment by enahum** [comment]
@jasonblais what we need to check is that both of the strings have the **same** key and that is indeed a bug, each string must have a **unique** key

**Comment by jasonblais** [comment]
Right, of course --

I'm pretty sure it should be 
```
{
    "id": "api.server.start_server.rate_limiting_memory_store",
    "translation": "Unable to initalize rate limiting memory store. Check MemoryStoreSize config setting."
  },
  {
    "id": "api.server.start_server.rate_limiting_rate_limiter",
    "translation": "Unable to initalize rate limiting."
  },
```

as `api.server.start_server.rate_limiting_rate_limiter` is called [here in server.go](https://github.com/mattermost/platform/blob/7fcc004beb9f6ef022f755e8e2f2a958c976c637/api/server.go#L90), where it would make sense to give the message `Unable to initalize rate limiting.`

**Comment by jasonblais** [comment]
Could we also update the strings here in this PR or would you prefer a separate one? At least to fix the `initalize --> initialize` typo

**Comment by enahum** [comment]
I think a separate PR is best

File: i18n/ja.json [hunk 10]
```diff
@@ -48,10 +48,6 @@
     "translation": "9月"
   },
   {
-    "id": "api.admin.add_certificate.array.app_error",
```
Comment (kaakaa) [inline i18n/ja.json]: This id is deleted in all languages.
But this id is used in [platform/admin\.go at 1e7985a87a72bea9a308cf1506dacc828c6e2e1c · mattermost/platform](https://github.com/mattermost/platform/blob/1e7985a87a72bea9a308cf1506dacc828c6e2e1c/api/admin.go#L661).

Is this ok?

Comment (enahum) [inline i18n/ja.json]: yes @kaakaa that's ok cause it was removed from `en.json` by mistake I think, I'll add it back for a future PR

Comment (kaakaa) [inline i18n/ja.json]: I got it! thank you @enahum!

**Comment by wget** [comment]
Ok for French. Don't forget to pull changes from the Pootle instance. I completed the French translation yesterday :-) Not a single English word remaining :-)

**Comment by jasonblais** [comment]
Thanks @wget! 

We'll include new and updated translations in the next translations pull request submitted to the `platform` repo.

**Comment by CometKim** [comment]
`ko` looks good to me :+1:

**Comment by crspeller** [comment]
@enahum Needs rebase.

**Comment by enahum** [comment]
@crspeller rebased

**Pull request merged**